NEWS.md

```diff
@@ -10,6 +11,27 @@
   not yet implemented in Spark.
   
 - Support for collecting NA values from logical columns (#1729).
+||||||| merged common ancestors
+<<<<<<<<< Temporary merge branch 1
```

Comment:
I believe you mentioned you are planning to fix this merge conflicts, just adding this note to make sure we don't forget.